### PR TITLE
Support

### PR DESCRIPTION
**Pull request by chenjianfei2017**
Support sdk release

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/support/azure-mgmt-support/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 0.2.0 (2020-03-14)
```
Comment (zikalino) [inline sdk/support/azure-mgmt-support/CHANGELOG.md]: changed version to 0.2.0, no need for 1.0.0

**Comment by zikalino** [comment]
@mit2nil pls not use "/azp run" here, it runs all the pipelines

**Comment by mit2nil** [comment]
@zikalino I tried that because checks were already failing because it was trying to run all pipelines. Either way, it didn't take that command as I don't have permission to run this. It looks like something wrong with pipeline setup itself. Why are checks still failing?

**Comment by zikalino** [comment]
merged as failing checks are invalid and unrelated

**Pull request merged**